From 572fd2a295de4237374166ee10773fcb412802c9 Mon Sep 17 00:00:00 2001
From: Duy Tran Khanh <40482367+khanhduytran0@users.noreply.github.com>
Date: Thu, 10 Jun 2021 08:46:35 +0700
Subject: [PATCH] Remove aarch64_get_thread_helper()

See commit 6f407090fda1 ("aarch64_get_thread_helper() ->
Thread::current()") for more details.
---
 src/hotspot/os_cpu/linux_aarch64/threadLS_linux_aarch64.S | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/hotspot/os_cpu/linux_aarch64/threadLS_linux_aarch64.S b/src/hotspot/os_cpu/linux_aarch64/threadLS_linux_aarch64.S
index f541844b9d6d..dd83b3723304 100644
--- a/src/hotspot/os_cpu/linux_aarch64/threadLS_linux_aarch64.S
+++ b/src/hotspot/os_cpu/linux_aarch64/threadLS_linux_aarch64.S
@@ -19,6 +19,7 @@
 // or visit www.oracle.com if you need additional information or have any
 // questions.
 
+#ifndef __ANDROID__
         // JavaThread::aarch64_get_thread_helper()
         //
         // Return the current thread pointer in x0.
@@ -42,3 +43,4 @@ _ZN10JavaThread25aarch64_get_thread_helperEv:
 	ret
 
 	.size _ZN10JavaThread25aarch64_get_thread_helperEv, .-_ZN10JavaThread25aarch64_get_thread_helperEv
+#endif
-- 
2.44.0

